packages/eas-build-job/src/metadata.ts

```diff
@@ -124,6 +124,16 @@ export type Metadata = {
    * Message attached to the build.
    */
   message?: string;
+
+  /**
+   * Indicates whether the build was run from CI.
+   */
+  runFromCI?: boolean;
```

Comment:
sounds like this is an option that enables running from CI, maybe trigeredFromCI, calledFromCI, ranFromCI, didRunFromCI?